docs/FAQ.md

```diff
@@ -0,0 +1,83 @@
+# Registry past-and-present cheat sheet
+
+> Everything You Always Wanted to Know about the Registry
+
+## What's a Registry?
+
+It's a server-side storage and delivery software that let you publish and exchange docker images.
+
+Whenever you do a `docker pull` or `docker push`, your engine communicates with a registry.
+
+## So, does the Docker Hub contain a registry?
+
+Yes.
+
+That "Hub registry" is operated by Docker.
+
+It is free to use for public content, and offers paid-for private storage as well.
+
+Support for the hub is handled directly by Docker, over here: https://support.docker.com
+
```

Comment:
Agreed, it is in need of a serious overhaul (especially now the other projects are included)

I do see questions regularly regarding "where's the source for Docker Hub", that's why I was thinking some short note could be meaningful.

But..these are all just quick "blurbs" from my side, so no problem